src/extension/core/src/main/java/au/org/emii/core/Config.java

```diff
@@ -0,0 +1,153 @@
+package au.org.emii.core;
```

Comment:
I remember talking about this, although I was thinking more about not refactoring the netcdfgenerator wps to use the same config code as the config there is completely different.  As a general rule duplicating code is horrible - especially a straight copy and paste.  Not sure when we will ever refactor ncwms to use it if not now.   Perhaps something to look at when we finish the gogoduck refactor.